bundles/core/src/main/java/com/adobe/cq/commerce/core/components/internal/models/v1/productlist/ProductListImpl.java

```diff
@@ -79,17 +71,30 @@
     private boolean showTitle;
     private MagentoGraphqlClient magentoGraphqlClient;
 
+    private int navPageCursor = 1;
+    private int navPageSize = 6;
```

Comment:
Don't set it to a hardcoded number, either use the default declared above or leave empty, will be assigned later anyway.